package.json

```diff
@@ -13,12 +18,17 @@
     "example": "examples"
   },
   "scripts": {
-    "generate-readme": "node bin/generate-readme.js",
-    "lint": "eslint lib examples bin",
-    "pretest": "npm run lint",
-    "test": "jest",
-    "coverage": "jest --coverage",
-    "start": "rackt server"
+    "generate-readme": "scripty",
```

Comment:
Never seen this module before, seems pretty ok if you're comfortable with it but it looks foreign to me to just have all the npm scripts point to a single `scripty` thing. Mostly just sounds like magic (which is kinda what rackt-cli was too, to me).